### PR TITLE
Fixes #6864 - adding api messages for extraction

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

**Comment by domcleal** [comment]
Did you mean to translate v1 files?  I ask because there are some you didn't do... we didn't extract other v1 strings in controllers.

**Comment by tstrachota** [comment]
Ok, I'll let v1 be then.

**Comment by domcleal** [comment]
Ok, thanks.  There are some error messages in app/controllers/api/base_controller.rb needing extraction too, e.g. not finding resources, or requiring an admin user.

**Comment by ohadlevy** [comment]
@isratrade can you review please?

File: app/controllers/api/base_controller.rb [hunk 1]
```diff
@@ -241,7 +241,13 @@ def find_nested_object
             # there should be a route error before getting here, but just in case,
             # throw an exception if a parameter 'xyz_id' was passed manually in URL string rather than generated by nested route.
             # unless it is explicitly declared in skip_nested_id in the case where there is 2-level nesting . Ex. puppetclasses/apache/smart_variables/4/override_values
-            raise "#{param} is not allowed as nested parameter for #{controller_name}. Allowed parameters are #{allowed_nested_id.join(', ')}" unless skip_nested_id.include?(param)
+            if !skip_nested_id.include?(param)
```
Comment (isratrade) [inline app/controllers/api/base_controller.rb:244]: why change from `unless skip_nested_id.include?(param)` to `if !skip_nested_id.include?(param)`?

**Comment by isratrade** [comment]
merged as 8f51acd4d6c83cc391217ebcb2ef2be54f920f20 with minor change from `if !` to `unless`. thanks @tstrachota !

**Pull request closed without merging**